*: adaptive analyze scan concurrency (#17894)
pingcap/docs: system-variables.md

- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
- Type: Integer
- Default value: `4`

> - Range: `[0, 4294967295]`. In versions prior to v8.2, the minimum value was 1. When set to 0, it will adapt the concurrency based on the cluster size.

Reviewer: ```suggestion
- Range: `[0, 4294967295]`. In versions prior to v8.2.0, the minimum value is `1`. When you set it to `0`, it adapts the concurrency based on the cluster size.
```

> - Range: `[0, 4294967295]`. In versions prior to v8.2.0, the minimum value is `1`. When you set it to `0`, it adapts the concurrency based on the cluster size.

Reviewer 2: ```suggestion
- Range: `[0, 4294967295]`. In versions earlier than v8.2.0, the minimum value is `1`. When you set it to `0`, it adaptively adjusts the concurrency based on the cluster size.
```

- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
- Type: Integer
- Default value: `1`

> - Range: `[0, 4294967295]`. The maximum value for v7.5.0 and earlier versions is `256`. Before v8.2.0, the minimum value is `1`. When you set it to `0`, it adapts the concurrency based on the cluster size.

Reviewer 2: ```suggestion
- Range: `[0, 4294967295]`. The maximum value for v7.5.0 and earlier versions is `256`. Before v8.2.0, the minimum value is `1`. When you set it to `0`, it adaptively adjusts the concurrency based on the cluster size.
```

> - Range: `[0, 4294967295]`. The maximum value for v7.5.0 and earlier versions is `256`. In versions prior to v8.2, the minimum value was 1. When set to 0, it will adapt the concurrency based on the cluster size.

Reviewer: ```suggestion
- Range: `[0, 4294967295]`. The maximum value for v7.5.0 and earlier versions is `256`. In versions prior to v8.2.0, the minimum value is `1`. When you set it to `0`, it adapts the concurrency based on the cluster size.
```

> - Range: `[0, 4294967295]`. The maximum value for v7.5.0 and earlier versions is `256`. In versions prior to v8.2.0, the minimum value is `1`. When you set it to `0`, it adapts the concurrency based on the cluster size.

Reviewer: ```suggestion
- Range: `[0, 4294967295]`. The maximum value for v7.5.0 and earlier versions is `256`. Before v8.2.0, the minimum value is `1`. When you set it to `0`, it adapts the concurrency based on the cluster size.
```
